go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/sygaldry/sygaldry
+
+go 1.13
+
+require (
```

Comment:
Is this an option for documenting dependencies that's different from the deps project?

Reply:
This makes deps unnecessary. It's autogenerated when you "go mod init" and then anything you "go get" gets added

Reply:
Does it add to a doc somewhere? I can add link to that in our docs! 😃

Reply:
I don't know about a doc, but it adds everything to the go.mod file